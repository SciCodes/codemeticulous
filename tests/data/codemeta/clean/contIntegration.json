{
  "@context": "https://w3id.org/codemeta/3.0",
  "@type": "SoftwareSourceCode",
  "name": "Software that uses CI",
  "contIntegration": "https://app.travis-ci.com/some/app"
}
