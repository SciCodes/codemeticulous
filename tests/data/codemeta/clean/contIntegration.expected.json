{
  "@context": "https://w3id.org/codemeta/3.0",
  "@type": "SoftwareSourceCode",
  "name": "Software that uses CI",
  "continuousIntegration": "https://app.travis-ci.com/some/app"
}
